packages/api/endpoints/dead-letter-archive.js

```diff
@@ -11,27 +11,39 @@ async function postRecoverCumulusMessages(req, res) {
   const stackName = process.env.stackName;
   const systemBucket = process.env.system_bucket;
 
-  const { bucket, path } = (req.body === undefined ? {} : req.body);
+  const {
+    bucket,
+    path,
+    batchSize = 1000,
+    concurrency = 30,
+    maxDbPool = 50,
+  } = req.body ?? {}; // TODO ZOD!  KNEEL BEFORE ZOD
   const asyncOperation = await asyncOperations.startAsyncOperation({
     cluster: process.env.EcsCluster,
     callerLambdaName: getFunctionNameFromRequestContext(req),
     lambdaName: process.env.DeadLetterProcessingLambda,
-    asyncOperationTaskDefinition: process.env.AsyncOperationTaskDefinition,
+    asyncOperationTaskDefinition: process.env.DeadLetterRecoveryTaskDefinition,
     description: 'Dead-Letter Processor ECS Run',
     operationType: 'Dead-Letter Processing',
     payload: {
+      batchSize,
       bucket,
+      concurrency,
       path,
     },
     stackName,
     systemBucket,
-    knexConfig: process.env,
+    knexConfig: { ...process.env, maxDbPool },
```

Comment:
why isn't maxDbPool being gotten originally from the environment? I note that further down in tests it's being configured as an environment variable